src/backend/server.ts

```diff
@@ -19,6 +20,13 @@ const BASE_PATH = '';
 const buildPath = path.resolve(process.cwd(), miljø.builldPath);
 const PORT = 3000;
 
+app.use(
+    RateLimit({
+        windowMs: 60 * 1000, // 60 seconds
+        limit: 120, // limit each IP to 120 requests per windowMs
```

Comment:
Har ikke alle saksbehandlere på 1 kontor den samme IP'n?

Reply:
Hvis man bruker en annen `keyGenerator` enn default så kan man sikkert bruke noe annet som verdi?

Reply:
@janeirik spurte om dette i tech-sikkerhet: 
> Alle trådløse enheter i NAV kommer fra 155.55.51.180 - 155.55.51.222

https://nav-it.slack.com/archives/C9SS0JU1X/p1702324300169499

Reply:
Ja, alle personer på NAV fordelt på 42 ip-adresser. Å bruke IP få føles vel feil?